src/api/categoriesApi.js

```diff
@@ -0,0 +1,10 @@
+import axiosClient from "./axiosClient";
+
+// api/categoiesApi.js
+const categoriesApi = {
+	getAll: (params) => {
+		const url = "/categories";
+		return axiosClient.get(url, { params });
```

Comment:
Mấy cái này gọi là repository hoặc services/

Nên có 1 cái base. Chứa các hàm common. ví dụ .getAll ( Thấy file nào cũng có )
```suggestion
// Base
export default class BaseService {
  constructor(resource) {
    this.resource = resource
  }
  
  getAll() {
      // code here ...
  }
}


// api/categoiesApi.js
import BaseService from './base'
import axiosClient from "./axiosClient";

export default class AnswerService extends BaseService {
  constructor(resource = 'answer_items') {
    super(resource)
  }
}

```